posts/2023-12-30_containers_webassembly_submission/containers_and_webassembly_submissions.qmd

```diff
@@ -0,0 +1,255 @@
+---
+title: "Testing Containers and WebAssembly in Submissions to the FDA"
+author:
+  - name: André Veríssimo
+  - name: Tymoteusz Makowski
+  - name: Pedro Silva
+  - name: Vedha Viyash
+  - name: APPSILON
+description: "In this post, we dig into the ongoing efforts undertaken to evaluate new technologies for submissions to the Food and Drug Administration (FDA). These transformative approaches, including WebAssembly and containers, hold immense potential to transform the regulatory landscape and streamline the submission process." # <---- please write a description
+date: "2023-12-30"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [submission, community]
+# feel free to change the image
+image: "FDA_WebR_Rhino.webp"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2023-12-30_containers_webassembly_submission"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+The R Consortium Submission Working Group has now successfully made two pilot submissions to the FDA. All the submissions done by the group are focused on improving practices for R-based clinical trial regulatory submissions. Now, the R submission Working Groups in collaboration with Appsilon and Posit are exploring new technologies such as Containers and WebAssembly. In this article, we dive into the details of this exploration.
+
+## How Everything Started
+
+### Pilot 1
+
```

Comment:
could we say what we mean by a publicly available submission?  like NN has down a full R-based submission, but it isn't available for us to look at (AFAIK) and others have done them (I've been told)